### PR TITLE
Fixed some old typos and re-worked some strings

### PR DESCRIPTION
**Pull request by MarkThat**
It would be cool to have these changes in the next update since the latest version has some outdated translations which may confuse users.

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -244,18 +244,39 @@
     <!-- debug strings -->
     <string name="debug_title">Impostazioni di debug</string>
     <string name="debug_leak_canary">LeakCanary</string>
-    <string name="app_restart">Prego, riavvia la app per applicare le modifiche.</string>
+    <string name="app_restart">Riavvia l'applicazione per applicare le modifiche.</string>
```
Comment (Roboe) [inline app/src/main/res/values-it/strings.xml]: Ouch. You forgot to escape the apostrophe. Build fails with the following error:
```
...values-it/values-it.xml:71: error: Apostrophe not preceded by \ (in Riavvia l'applicazione per applicare le modifiche.)
```

**Comment by Roboe** [comment]
(Also, you don't need to include languages strings, as those depends on #553 and are already included there)

**Comment by MarkThat** [comment]
Ops, yeah! I will fix it asap.

**Comment by MarkThat** [comment]
Sorry for the confusion with commits. I need to get some sticky notes to start remember things before pushing them :P

**Comment by anthonycr** [comment]
Awesome!

**Pull request merged**